src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -2306,8 +2306,6 @@ static double accSymId2alter(SymId id)
         break;
     case SymId::accidental4CommaFlat:                      res = -0.89;
         break;
-    case SymId::accidentalSharp:                           res =  0.89;
-        break;
```

Comment:
Maybe leave that in but commented out, like in accidental.cpp?

Reply:
Except for consistency. Maybe also leave a comment in this file and in accidentals.cpp, reg. the need to keep them in sync